jsaddle-warp/src/Language/Javascript/JSaddle/WebSockets.hs

```diff
@@ -189,15 +189,15 @@ jsaddleJs' jsaddleUri refreshOnLoad = jsaddleCoreJs <> "\
     \    };\n\
     \    var core = jsaddle(window, function(a) {\n\
     \      ws.send(JSON.stringify(a));\n\
-    \    }, sync);\n\
+    \    }, sync, 10 /* RESPONSE_BUFFER_MAX_SIZE (0 to disable) */);\n\
     \    var syncKey = \"\";\n\
     \\n\
     \    ws.onopen = function(e) {\n\
     \\n\
     \        ws.onmessage = function(c) {\n\
     \            connId = c.data;\n\
     \            ws.onmessage = function(e) {\n\
-    \                core.processReq(JSON.parse(e.data));\n\
+    \                core.processReqs(JSON.parse(e.data));\n\
     \            };\n\
     \        }\n\
     \    };\n\
```

Comment:
Dumb idea but I see no reason to keep these `\n` around. They aren't needed in the output anyway.

Reply:
I think they may help with debugging this code in-browser.

Reply:
Most browsers include a code prettifier when debugging.